functions/bfc-functions.php

```diff
@@ -124,6 +124,121 @@ function bfc_nouveau_has_nav( $args = array() ) {
+    'home', // parent top nav to highlight
+    '/', // page, use relative urls
+    '/wp-content/themes/bfcom/assets/images/home.svg', // icon          // menu item icon
+    'Home' // menu item text
+  ),
+  array (
+    'bfc-topnav-members',
+    'members menu-item-type-post_type menu-item-object-page',
+    'members',
+    '/members/',
+    '/wp-content/themes/bfcom/assets/images/members.svg',
+    'People'
+  ),
+  array (
+    'bfc-topnav-groups',
+    'groups menu-item-type-post_type menu-item-object-page',
+    'groups',
+    '/groups/',
+    '/wp-content/themes/bfcom/assets/images/groups.svg',
+    'Groups'
+  ),
+  array (
+    'bfc-topnav-resources',
+    'resources menu-item-type-post_type menu-item-object-page',
+    'resources',
+    '/wiki/',
+    '/wp-content/themes/bfcom/assets/images/resources.svg',
+    'Resources'
+  ),
+  array (
+    'bfc-topnav-blogs',
+    'blogs menu-item-type-post_type menu-item-object-page',
+    'blogs',
+    '/sites/',
+    '/wp-content/themes/bfcom/assets/images/blog.svg',
+    'Blogs'
+  ),
+  array (
+    'bfc-topnav-search',
+    'search menu-item-type-post_type menu-item-object-page',
+    'search',
+    '/activity/',
+    '/wp-content/themes/bfcom/assets/images/search.svg',
+    'Search'
+  )
+);
+return $menuitemarray;
+}
+
+function bfc_top_nav() {
+  // Build collection for this template method
+  $menuitemarray = bfc_top_nav_menu_builder();
+
+  // set $topnav to its initial value
+    $topnav = '<ul id="bfc-topmenu" class="medium-horizontal menu">'; //start with the opening ul and its selectors
+
+  // loop through each menu item in the collection to build topnav html
+  foreach ($menuitemarray as $menuitem) {
+    $thismenuitem =  '<li '; // start the li
+    $thismenuitem .= 'id="';
+    $thismenuitem .= $menuitem[0]; // add id this menu item
+    $thismenuitem .= '" class="menu-item '; // common selectors for menu item
+    $thismenuitem .= $menuitem[1]; // add any specific selectors for this menu item
+
+    if (bfc_top_nav_is_active($menuitem[2])) {
+          $thismenuitem .= ' active'; // highlight the active parent
+    }
+    $thismenuitem .= '" role="menuitem"><a href="';
+      $thismenuitem .= $menuitem[3]; // add link
+    $thismenuitem .= '">';
+    $thismenuitem .= '<img src="';
+    $thismenuitem .= $menuitem[4]; // add icon
+    $thismenuitem .= '">';
+    $thismenuitem .= $menuitem[5]; // menu item text
+    $thismenuitem .= '</a></li>'; // close off the menu item
+
+    $topnav .= $thismenuitem; // add menu item to nav
+  }
+    $topnav .= '</ul>'; //close ul
+
+    echo $topnav;
+}
+
+// determine of the top menu item should be highlighted
+function bfc_top_nav_is_active($topmenuparent) {
+
+ if ($topmenuparent == 'home' && is_front_page()) {
+        return true;
+    }
+  else if ($topmenuparent == 'members' && (is_page('members') || bp_is_user())) {
```

Comment:
The coding style guide suggests using`elseif` instead of `else if`. I don't know that it's a big deal for us, but we might as well get in the habit.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#use-elseif-not-else-if

Reply:
Fixed